modules/apps/depot/depot-web/src/main/resources/META-INF/resources/js/DepotAdminManagementToolbarPropsTransformer.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {openSimpleInputModal, postForm} from 'frontend-js-web';
+
+import confirmDepotEntryDeletion from './confirmDepotEntryDeletion.es';
+
+export default function propsTransformer({
+	additionalProps: {deleteDepotEntriesURL},
+	portletNamespace,
+	...otherProps
+}) {
+	const deleteSelectedDepotEntries = () => {
+		if (confirmDepotEntryDeletion()) {
+			const form = document.getElementById(`${portletNamespace}fm`);
+
+			if (!form) {
+				return;
+			}
+
+			postForm(form, {
+				data: {
+					deleteEntryIds: Liferay.Util.listCheckedExcept(
+						form,
+						`${portletNamespace}allRowIds`
+					),
+				},
+				url: deleteDepotEntriesURL,
+			});
+		}
+	};
+
+	return {
+		...otherProps,
+		onActionButtonClick: (event, {item}) => {
+			if (item.data?.action === 'deleteSelectedDepotEntries') {
+				deleteSelectedDepotEntries();
+			}
+		},
+		onCreateButtonClick: (event, {item}) => {
+			openSimpleInputModal({
+				dialogTitle: Liferay.Language.get('add-asset-library'),
+				formSubmitURL: item.data.addDepotEntryURL,
```

Comment:
Is this item data is always present?

Reply:
Hey @boton,

I believe the item.data is always present since this is being added via the backend (see https://github.com/liferay-lima/liferay-portal/blob/1d78a472a4e29f44161d8b03271ccf27b3551993/modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/display/context/DepotAdminManagementToolbarDisplayContext.java#L126...L127) but as stated before, it can be an issue if that was removed.  I will add a null check just to be safe.  

Thanks!

Reply:
Pushed changes at https://github.com/liferay-lima/liferay-portal/pull/1478/commits/b80bbde49b7b06f0959c8f80398ed81f810293b2#diff-c8b4e06b731aa7ba47ab1a31d8a8a285da862210934efb6d3606700b1fd9c18dR54

Reply:
Thanks!